arkouda/dataframe.py

```diff
@@ -1248,7 +1247,16 @@ def save_table(self, prefix_path, columns=None, index=False, file_format='HDF5')
 
     @classmethod
     def load_table(cls, prefix_path, file_format='INFER'):
-        return cls(load_all(prefix_path, file_format=file_format))
+        prefix, extension = os.path.splitext(prefix_path)
+        first_file = "{}_LOCALE0000{}".format(prefix, extension)
+        filetype = get_filetype(first_file)
+        # columns load backwards
+        df = cls(load_all(prefix_path, file_format=filetype))
+        if filetype == "HDF5":
+            return df
+        else:
```

Comment:
What about explicitly checking the filetype, then raising an exception if it is not "HDF5" or "Parquet"? This might save trouble down the road if some other filetype is added... Mostly pointing out because there have been other bugs caused by similar assumptions...
```
elif filetype == "Parquet":
            # return the dataframe with them reversed so they match what was saved. This is only an issue with parquet
            return df[df.columns[::-1]]
else:
            #raise some appropriate exception...
```

Reply:
@pierce314159 & @kellyjoy15 I initially had it that way, but it seemed redundant to do the same check again. @kellyjoy15 do you have any issues with leaving this?